llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/TileAndDecomposeAttention.cpp

```diff
@@ -176,33 +194,17 @@ extractSlices(Value key, Value value, Value query, Value output,
       loc, tensorType, value, offsets, sizes, strides);
 
   offsets = SmallVector<OpFoldResult>(queryShape.size(), zero);
-  offsets[0] = ivs[0];
   Value querySlice = builder.create<tensor::ExtractSliceOp>(
       loc, tensorType, query, offsets, sizes, strides);
-  Value outputSlice = builder.create<tensor::ExtractSliceOp>(
-      loc, tensorType, output, offsets, sizes, strides);
-  return std::make_tuple(keySlice, valueSlice, querySlice, outputSlice);
+  return std::make_tuple(keySlice, valueSlice, querySlice);
 }
 
 static std::tuple<Value, Value, Value>
 insertSlices(Value newResult, Value result, Value newMax, Value max,
              Value newSum, Value sum, ArrayRef<int64_t> queryShape,
              ArrayRef<Value> ivs, OpFoldResult sequenceTileLength,
              OpFoldResult headDimension, Location loc, OpBuilder &builder) {
-  auto one = builder.getIndexAttr(1);
-  auto zero = builder.getIndexAttr(0);
-  SmallVector<OpFoldResult> strides(queryShape.size(), one);
-  SmallVector<OpFoldResult> sizes(queryShape.size(), one);
-  SmallVector<OpFoldResult> offsets(queryShape.size(), zero);
-  sizes[1] = sequenceTileLength;
-  sizes[2] = headDimension;
-  offsets[0] = ivs[0];
-  Value updatedAcc = builder.create<tensor::InsertSliceOp>(
-      loc, newResult, result, offsets, sizes, strides);
-  offsets = SmallVector<OpFoldResult>(queryShape.size() - 1, zero);
-  sizes = SmallVector<OpFoldResult>{one, sequenceTileLength};
-  strides = SmallVector<OpFoldResult>(queryShape.size() - 1, one);
-  return std::make_tuple(updatedAcc, newMax, newSum);
+  return std::make_tuple(newResult, newMax, newSum);
 }
```

Comment:
Do we still need this function...? It looks very redundant.

Reply:
Thanks, will remove this function.